pydoctor/templatewriter/summary.py

```diff
@@ -183,7 +183,7 @@ def link(obj: model.Documentable) -> Tag:
             # category (class, method, etc.) an object belongs to.
             tag: Tag = tags.code(
                 epydoc2stan.taglink(obj, NameIndexPage.filename),
-                **{"data-type": obj.kind}
+                **{"data-type": epydoc2stan.format_kind(obj.kind) if obj.kind else ''}
```

Comment:
Can the `kind` be `None` at this stage? If not, you can assert that it is not `None`. If it can be `None`, maybe it's better to omit the `data-type` attribute than to add it with an empty value.

Reply:
Ok, in any case the doc2dash parser needs an update and will use intersphinx so this doesn't really matter anymore.